bot/exts/utilities/bookmark.py

```diff
@@ -57,6 +96,17 @@ def build_error_embed(message: str) -> discord.Embed:
             colour=Colours.soft_red
         )
 
+    @staticmethod
+    def create_button_embed(target_message: discord.Message) -> discord.Embed:
+        """Generates the embed which is sent with the button, so other users can receive the bookmark too."""
+        return discord.Embed(
+            description=(
+                f"Click the button to be sent your very own bookmark to "
+                f"[this message]({target_message.jump_url})."
+            ),
+            colour=Colours.soft_green,
+        )
+
```

Comment:
I'm not sure this needs to be it's own function anymore, since it's just creating a single embed, and it's only used in one place, right at the end of a function, so doesn't harm readibility.